src/main/webapp/aboutus.jsp

```diff
@@ -41,8 +41,123 @@ limitations under the License.
+    function addLandmark(map, lat, lng, title, description){ 
+      const marker = new google.maps.Marker({ 
+        position: {lat: lat, lng: lng}, 
+        map: map, 
+        title: title 
+      }); 
+      const infoWindow = new google.maps.InfoWindow({ 
+        content: description 
+      }); 
+      marker.addListener('click', function() { 
+        infoWindow.open(map, marker); 
+      }); 
+    }
+    /** Creates a marker that shows a read-only info window when clicked. */
+    function createMarkerForDisplay(lat, lng, content){
+      const marker = new google.maps.Marker({
+        position: {lat: lat, lng: lng},
+        map: map
+      });
+      var infoWindow = new google.maps.InfoWindow({
+        content: content
+      });
+      marker.addListener('click', () => {
+        infoWindow.open(map, marker);
+      });
+    }
+    /** Sends a marker to the backend for saving. */
+    function postMarker(lat, lng, content){
+      const params = new URLSearchParams();
+      params.append('lat', lat);
+      params.append('lng', lng);
+      params.append('content', content);
+      fetch('/markers', {
+        method: 'POST',
+        body: params
+      });
+    }
+    /** Creates a marker that shows a textbox the user can edit. */
+    function createMarkerForEdit(lat, lng){
+      // If we're already showing an editable marker, then remove it.
+      if(editMarker){
+       editMarker.setMap(null);
+      }
+      editMarker = new google.maps.Marker({
+        position: {lat: lat, lng: lng},
+        map: map
+      });
+      const infoWindow = new google.maps.InfoWindow({
+        content: buildInfoWindowInput(lat, lng)
+      });
+      // When the user closes the editable info window, remove the marker.
+      google.maps.event.addListener(infoWindow, 'closeclick', () => {
+        editMarker.setMap(null);
+      });
+      infoWindow.open(map, editMarker);
+    }
+    /** Builds and returns HTML elements that show an editable textbox and a submit button. */
+    function buildInfoWindowInput(lat, lng){
+      const textBox = document.createElement('textarea');
+      const button = document.createElement('button');
+      button.appendChild(document.createTextNode('Submit'));
+      button.onclick = () => {
+        postMarker(lat, lng, textBox.value);
+        createMarkerForDisplay(lat, lng, textBox.value);
+        editMarker.setMap(null);
+      };
+      const containerDiv = document.createElement('div');
+      containerDiv.appendChild(textBox);
+      containerDiv.appendChild(document.createElement('br'));
+      containerDiv.appendChild(button);
+      return containerDiv;
+    }
+  </script>
+  <style>
+    #map{
+      width: 500px;
+      height: 500px;
+    }
+  </style>
   </head>
-  <body>
+  <body onload="createMap();">
+  <h1>Team 1 Map</h1>
+  <div id="map"></div>
+  <p>Find out where Team 1 is, or add your own personal marker to the map!</p>
```

Comment:
Maybe include another sentence explaining what to do? Something like: "Click in the map and enter your location to save your hometown"?